api/v1/groupversion_info.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 // Package v1alpha1 contains API Schema definitions for the datasciencepipelinesapplications v1alpha1 API group
 // +kubebuilder:object:generate=true
 // +groupName=datasciencepipelinesapplications.opendatahub.io
-package v1alpha1
+package v1
```

Comment:
under the hood dspo will update them automaticaly to v1, when doing a `oc get` or viewing on the console, you will get either `v1` or `v1alpha1` set for `apiversion` depending on what path you use, but otherwise they will look the same